test/integration/model.test.js

```diff
@@ -844,6 +844,32 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         .then(() => User.findByPk(94))
         .then(user => expect(user.id2).to.equal(8877));
     });
+
+    it('should use PK attribute DataType', function() {
```

Comment:
Rename this to something more clear, such as:

```suggestion
    it('should call stringify from PK datatype correctly upon save', function() {
```